backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/UserService.java

```diff
@@ -134,4 +134,25 @@ private Predicate<ConfirmationCodeEntity> byCode(String code) {
 		return confirmationCode -> confirmationCode.getCode().equals(code);
 	}
 
+	private Predicate<ConfirmationCodeEntity> notExpired() {
+		return confirmationCode -> confirmationCode.getExpiryDate().isAfter(Instant.now());
+	}
+
+    public Boolean verifyConfirmationCode(String code, User user){
```

Comment:
Pass in the confirmation code and the user id. Don't force the caller to construct a `User` object when a simple string would be sufficient.

Reply:
Makes sense, implemented in commit 2cebd2d6f0685660fa512c66c4a5da16869070e1.